docs/connecting-data/connecting-to-apis-and-external-data/asana-api.md

```diff
@@ -1,7 +1,177 @@
----
-description: How to Connect DataDistillr to the Asana API
----
+# Connecting to the Asana API
 
-# Asana API
+## Creating an Asana account
+Set up an account with [Asana](https://asana.com/create-account).
 
-Details coming soon!
\ No newline at end of file
+### **Costs**
+There are a few different types of accounts:
+
+- Free
+- $10.99 per user per month billed annually or $13.49 billed monthly
+- $24.99 per user per month billed annually or $30.49 billed monthly
+
+You can check [HERE](https://asana.com/pricing) for details and comparison.
+
+### **Rate Limits**
+Limits are allocated per authorization token. Limits may vary from token to token.
+
+Free account limit: 150 requests per minute.
+Premium account limit: 1500 requests per minute.
+SearchAPI  are limited to 60 requests per minute.
+Duplication endpoints are limited to 5 concurrent jobs.
+
+## How to Connect DataDisillr to Asana
+To set up a data source connection for Asana, you will need to have:
+
+- A unique name for your data source connection to be used in queries.
```

Comment:
For consistency, link 'unique name' and 'API key' to the sections below like is done in the JIRA form.